medmodels/medrecord/_overview.py

```diff
@@ -0,0 +1,182 @@
+from typing import Dict, List, Optional, Union
+
+import polars as pl
+
+from medmodels.medrecord.schema import AttributesSchema, AttributeType
+from medmodels.medrecord.types import Attributes, EdgeIndex, NodeIndex
+
+
+def extract_attribute_summary(
+    attribute_dict: Union[Dict[EdgeIndex, Attributes], Dict[NodeIndex, Attributes]],
+    schema: Optional[AttributesSchema] = None,
+    decimal: int = 2,
+) -> pl.DataFrame:
+    """Extracts a summary from a node or edge attribute dictionary.
+
+
+    Example:
+    ┌────────────────┬──────────────────────────┐
```

Comment:
example is wrong, no?

Reply:
why? This is not for the printed table, but the data frame for each individual group.